app/components/Dashboard/BestNewMusicTab/BestNewMusicMenu/BestNewMusicCard/index.js

```diff
@@ -24,24 +30,94 @@ function toFavoriteTrack({ artist, title, thumbnail }) {
   };
 }
 
+function getReleaseInfo({ artist, title }) {
+  let masterId;
+  return findAlbum(artist, title)
```

Comment:
We don't need this here. Best new music card should only be concerned with presentation, and the minimal amount of logic. To add a favorite album, you only need what's already available from Pitchfork, i.e. artist, name, and thumbnail. Everything else will be looked up when we click the album.

If memory fails me though, and it doesn't work that way, then it'll be worthwhile to move this piece of logic to the favorite albums container, so that the discogs lookup is performed upon clicking the album. We really only need the id, so that we know where to go, but soon I might be changing that so we end up only looking up stuff after we go to the album page.